daemons/fenced/fenced_commands.c

```diff
@@ -352,7 +352,7 @@ stonith_device_execute(stonith_device_t * device)
         if (pending_op && pending_op->delay_id) {
             crm_trace
                 ("Operation %s%s%s on %s was asked to run too early, waiting for start_delay timeout of %dms",
-                 pending_op->action, pending_op->victim ? " for node " : "",
+                 pending_op->action, pending_op->victim ? " targeting " : "",
```

Comment:
These are clearer, thanks. Again not worth changing but just a comment: I find it's also helpful to quote fencing actions, because logs like "Performing on on node1" are difficult to read vs. "Performing 'on' on node1" or "Performing 'on' action on node1".

Reply:
Makes sense. We could as well take the opportunity to change as many of them.